Source/SPUD/Private/SpudSubsystem.cpp

```diff
@@ -1329,6 +1329,13 @@ void USpudStreamingLevelWrapper::OnLevelHidden()
 		{
 			spud->PreUnloadStreamingLevel.Broadcast(FName(levelName));
 			spud->HandleLevelUnloaded(level);
+
+			if (!spud->StreamLevelUnloadTimerHandle.IsValid())
+			{
+				// since there was no stream request added, there's no unloaded timer, so the notification will not be
+				// sent
+				spud->PostUnloadStreamingLevel.Broadcast(FName(levelName));
```

Comment:
I can't accept this change because a) testing that handle is kind of unrelated and brittle and b) claiming the level has been unloaded when it hasn't is incorrect and will only cause problems later.

If you needed some sort of extra notification then it should be done in a more "honest" way ;)